README.md

```diff
@@ -36,6 +36,18 @@ npm install
 npm run dev
 ```
 
+## Try with Docker
```

Comment:
Hi @codenuru could you split this “Docker” section in two? One for the development environment, and one for production build (with this disclaimer), I believe that with these two section will be more clear for those that want to use the template with Docker.

Reply:
Hi @raisiqueira I have made the changes, I hope it is clearer.

Reply:
Nice, thanks!